### ISSUE TITLE
selected text drops the NewLine \n

### ISSUE THREAD

**Issue by jpdup** [adrianjagielak/flutter_markdown_selectionarea#5]
It seems like the \n newlines are dropped when looking at the return plainText from 

`onSelectionChanged: (SelectedContent? textSelection)`

is there way to retain the new lines?

**Comment by huanguan1978** [comment]
<img width="454" alt="Screenshot 2024-07-16 at 11 57 40" src="https://github.com/user-attachments/assets/dac76193-0fcb-43d3-8bd6-a25e84c58536">
<img width="1000" alt="Screenshot 2024-07-16 at 11 59 44" src="https://github.com/user-attachments/assets/a88f449b-888d-4602-996a-782b59c8858b">

Copy all the text and paste it, and the line breaks are lost. 
I also found the same problem.

**Comment by Juliotati** [comment]
@jpdup the original package [flutter_markdown](https://pub.dev/packages/flutter_markdown) might have fixed the issue this package addressed before. Try it out.

**Comment by shubhang21** [comment]
@huanguan1978 Have you found any fix for this? Facing the same issue and looking for workarounds.

**Comment by huanguan1978** [comment]
@shubhang21 I couldn't find a solution, but I've now bypassed the problem using the share text method from the share_plus package.

**Comment by adrianjagielak** [comment]
@shubhang21 @huanguan1978 you could also add a "copy to clipboard" button and use `Clipboard.setData(ClipboardData(text: "text"));` (a built-in Flutter method) for it.

**Comment by huanguan1978** [comment]
@adrianjagielak  Yes, you are correct, but I need to share it with others, so the "share" method is better for me.

**Comment by adrianjagielak** [comment]
@huanguan1978 makes sense

**Comment by bryantwilliam** [comment]
See my fixes: https://github.com/flutter/flutter/issues/99819#issuecomment-2541138333
